### PR TITLE
add further instruction to get have the project running, failures due…

### PR DESCRIPTION
**Pull request by JonLazariniDD**
… to selectors changed

add clarfication for project builder maven

### CODE REVIEW

**Review by githanium — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+src/main/resources/conf/**
+src/test/resources/serenity.conf
+src/test/resources/webdriver/**
+src/test/resources/serenity.properties
+.env*
+target
+*.iml
+serenity.properties
+# ommitting project config as it has to be agreed with the team
+.idea
```
Comment (IuliaDVSA) [inline .gitignore:10]: Please add a blank line at the end of the doc.

File: src/test/java/pages/LoginPage.java [hunk 2]
```diff
@@ -15,12 +15,12 @@
     private static final String PASSWORD_INPUT = "input[type='password']";
     private static final String SIGN_IN = "input[type='submit']";
     private static final String HEADER = "div[role='heading']";
-    private static final String OPTIONS = "#idA_PWD_SwitchToCredPicker";
+//    private static final String OPTIONS = "#idA_PWD_SwitchToCredPicker";
```
Comment (IuliaDVSA) [inline src/test/java/pages/LoginPage.java:18]: Please remove commented out code.

File: src/test/java/pages/LoginPage.java [hunk 3]
```diff
@@ -15,12 +15,12 @@
...
 
     public void inputEmail(String emailAddress) {
         getDriver().manage().window().maximize();
         FluentWait wait = globalFluentWait(10, 250);
-        wait.until(ExpectedConditions.textToBePresentInElement(findElementByCss(OPTIONS), "Sign-in option"));
+//        wait.until(ExpectedConditions.textToBePresentInElement(findElementByCss(OPTIONS), "Sign-in option"));
```
Comment (IuliaDVSA) [inline src/test/java/pages/LoginPage.java:23]: Please remove commented-out code

File: .gitignore (hunk 1 above)
Comment (JonLazariniDD) [reply .gitignore:10]: I will leave it as it is for and let the tester team decide how they wish to format the code repo with static analysis tool or further config (.editorconfig, etc.. for example) as I am not maintaining this and believe should be a wider discussion than just a single developer/tester. Resolving

**Review by m-ujaffer — DISMISSED** [review]

**Comment by JonLazariniDD** [comment]
changes done and resolved @IuliaDVSA, good catch thanks

**Review by m-ujaffer — APPROVED** [review]

**Review by githanium — APPROVED** [review]

**Pull request merged**